### PR TITLE
TF migration docs - round 4

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/test-fairy/testing-an-app/testers/managing-testers.md [hunk 1]
```diff
@@ -8,7 +8,40 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+In the list box, add tester emails - one for each row.
+
+You can also select a Group for the testers in the list or leave it blank (see **Managing tester groups** below).
+After you finish, click **Add Testers** below the list to complete the process.
+
+You can add testers manually or [import lists of testers](https://app.testfairy.com/testers/import/) in CSV format.
+
+:::note Only for iOS
+If you are **not** using an [iOS Enterprise certificate](https://developer.apple.com/programs/ios/enterprise/), you must get the UDIDs of your testers' devices before sending them your app.
+When you invite new testers by email, your testers will get an email asking them to register their device. Once they click on the registration link, you will get an email with their UDID, and their device details will be added to your [testers page](https://app.testfairy.com/testers).
```
Comment (LoryApiFortress) [inline docs/test-fairy/testing-an-app/testers/managing-testers.md:27]:
```suggestion
When you invite new testers by email, your testers get an email asking them to register their device. Once they click on the registration link, you will get an email with their UDID, and their device details will be added to your [testers page](https://app.testfairy.com/testers).
```

File: docs/test-fairy/testing-an-app/testers/managing-testers.md [hunk 2]
```diff
@@ -8,7 +8,40 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+For more information about how to add UDIDs to provisioning profiles, read [this guide](/test-fairy/sdk/ios/adding-udids).
+:::
+
+<img src={useBaseUrl('/img/test-fairy/testing-an-app/invite-testers2.png')} alt="alt upload"/>
+
+## Managing Tester Groups
+
+You can also divide testers into **groups** to add more structure and organize your testing efforts.
+To create a group, click in the GROUPS text box and then on `Create new group...`
+Choose a group name and click OK. The tester you clicked in his group will be assigned the group you created.
```
Comment (LoryApiFortress) [inline docs/test-fairy/testing-an-app/testers/managing-testers.md:37]:
```suggestion
Choose a group name and click OK. The tester you clicked in their group will be assigned the group you created.
```

File: docs/test-fairy/testing-an-app/testers/testers-dashboard.md [hunk 3]
```diff
@@ -8,7 +8,49 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+The [Testers Dashboard](https://my.testfairy.com/) displays the apps a tester was invited to test, and is viewable by all testers in the system.
```
Comment (LoryApiFortress) [inline docs/test-fairy/testing-an-app/testers/testers-dashboard.md:11]:
```suggestion
The [Testers Dashboard](https://my.testfairy.com/) displays the apps a tester is invited to test and is viewable by all testers in the system.
```

File: docs/test-fairy/testing-an-app/testers/user-feedback.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+When the TestFairy SDK https://docs.testfairy.com/SDK/Adding_The_SDK_To_Your_App.html is defined in your app, shaking the device will prompt the user to create a feedback issue from within the app.
```
Comment (LoryApiFortress) [inline docs/test-fairy/testing-an-app/testers/user-feedback.md:11]:
```suggestion
When the TestFairy SDK https://docs.testfairy.com/SDK/Adding_The_SDK_To_Your_App.html is defined in your app, shaking the device will prompt the user to create a feedback issue from in the app.
```

File: docs/test-fairy/using-test-fairy/builds.md [hunk 5]
```diff
@@ -8,5 +8,53 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+- **SESSIONS** - The number of sessions logged for the build. The number is linked to the **INSIGHTS** tab.
+
+- **CRASHES** - The number of logged crashes for the build version. The number is linked to the **CRASHES** tab.
+
+- **DOWNLOADS** - The number of downloads for the build.
+
+- **FEEDBACKS** - The amount of feedback received for the build. The number is linked to the **FEEDBACK** tab.
+
+- **UPLOADED** - When the was build uploaded.
```
Comment (LoryApiFortress) [inline docs/test-fairy/using-test-fairy/builds.md:25]:
```suggestion
- **UPLOADED** - When the build is uploaded.
```

File: docs/test-fairy/using-test-fairy/builds.md [hunk 6]
```diff
@@ -8,5 +8,53 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+### Metadata
+
+Metadata is specific information about a build. It is defined when the build is uploaded via Upload API and cannot be changed after it is uploaded.
+
+The metadata format is **metadata.key=value** (for example, `metadata.branch=master`). Only the value is searchable in the `Search` box at the top of the **Builds** table.
+
+### Deleting Builds and Apps
+
+To delete a build from the system, select the checkbox and, in the **More Actions…** dropdown, click **Delete Build**.
```
Comment (LoryApiFortress) [inline docs/test-fairy/using-test-fairy/builds.md:58]:
```suggestion
To delete a build from the system, select the checkbox and, in the **More Actions…** dropdown list, click **Delete Build**.
```

File: docs/test-fairy/using-test-fairy/builds.md [hunk 7]
```diff
@@ -8,5 +8,53 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+
+To delete an app, select all of its builds and, in the **More Actions…** dropdown, click **Delete Build**.
```
Comment (LoryApiFortress) [inline docs/test-fairy/using-test-fairy/builds.md:60]:
```suggestion
To delete an app, select all of its builds and, in the **More Actions…** dropdown list, click **Delete Build**.
```

File: docs/test-fairy/using-test-fairy/dashboard.md [hunk 8]
```diff
@@ -8,5 +8,47 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+The **CRASHES** tab displays crashes aggregated by stack trace. You can filter the list by app, version, and timeframe.
+Each crash is aggregated after symbolication (if possible) and appears on a separate line. Unsymbolicated crashes may appear in different lines, one for each crash.
+
+### **USER FEEDBACK** Tab
+
+<img src={useBaseUrl('/img/test-fairy/using-tf/dashboard-feedbacks.png')} alt="dashboard feedbacks"/>
+
+The **USER FEEDBACK** tab displays user feedback submitted using the shake-to-report feature. The feedback is linked to the session in which it was created and contains: - A screenshot. - The text the user entered. - The reporter's email (which can be different from the user identified in the session).
+
+Each issue can be linked to a bug tracking system by clicking the **CREATE BUG** button in the **ISSUE** column. Apps already connected to a bug-tracking system will display the issue link/number inside the button.
```
Comment (LoryApiFortress) [inline docs/test-fairy/using-test-fairy/dashboard.md:54]:
```suggestion
Each issue can be linked to a bug tracking system by clicking the **CREATE BUG** in the **ISSUE** column. Apps already connected to a bug-tracking system will display the issue link/number inside the button.
```

File: docs/test-fairy/testing-an-app/testers/user-feedback.md [hunk 9]
```diff
@@ -8,7 +8,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+When the TestFairy SDK https://docs.testfairy.com/SDK/Adding_The_SDK_To_Your_App.html is defined in your app, shaking the device will prompt the user to create a feedback issue from in the app.
```
Comment (LoryApiFortress) [inline docs/test-fairy/testing-an-app/testers/user-feedback.md:11]: What should I do with the link? It is not mentioned in the spreadsheet

**Review by aoleszyn — APPROVED** [review]

**Pull request merged**